dsc_lib/src/dscresources/command_resource.rs

```diff
@@ -146,24 +143,21 @@ pub fn invoke_set(resource: &ResourceManifest, cwd: &str, desired: &str, skip_te
 
     let mut get_env: Option<HashMap<String, String>> = None;
     let mut get_input: Option<&str> = None;
-    let mut get_args = resource.get.args.clone();
+    let args = process_args(&resource.get.args, desired);
     match &resource.get.input {
```

Comment:
would it be simpler to add this `match` block to `process_args` or as its own function since it's repeated within the other commands?

Reply:
This function is only for processing args, but you're right there's some redundancy.  Let me pull out the common part as a function.